src/cbc_sdk/platform/__init__.py

```diff
@@ -5,6 +5,8 @@
 from cbc_sdk.platform.alerts import (BaseAlert, WatchlistAlert, CBAnalyticsAlert, DeviceControlAlert,
                                      ContainerRuntimeAlert, Workflow, WorkflowStatus)
 
+from cbc_sdk.platform.asset_groups import AssetGroup
```

Comment:
In the file docs/cbc_sdk.platform.rst could this snippet be added please? (Ack that rtd would have come along in a later PR, but this will get us the doc strings along the way. Thanks!)
I've configured rtd to generate the feature-asset-groups branch so anyone who wants to use the feature-stable functions will have matching docs.

cbc\_sdk.platform.asset_groups module
-------------------------------------

.. automodule:: cbc_sdk.platform.asset_groups
   :members:
   :undoc-members:
   :show-inheritance:

Reply:
That wasn't part of the requirements; maybe we need to make sure it is for any future set of stories that includes a new module.  It's being added.